python/kata/combine.py

```diff
@@ -1,8 +1,11 @@
 def combine_together(d1, d2):
-    """ Write a code to combine two dictionary adding values for common keys. """
-
 
+    for i in d2.keys():
+        if i in d1:
+            d2[i] = d2[i] + d1[i]
+          
 d1 = {'a': 100, 'b': 200, 'c':300}
-d2 = {'a': 300, 'b': 200, 'd':400}
```

Comment:
Please don't change anything.
We have these two dictionaries and we want to combine them.
d1 = {'a': 100, 'b': 200, 'c':300}
d2 = {'a': 300, 'b': 200, 'd':400}
We also want the "d" key, so do not change it.